nautobot_floor_plan/filter_extensions.py

```diff
@@ -2,12 +2,13 @@
 
 import django_filters
 from nautobot.apps.filters import RelatedMembershipBooleanFilter
-from nautobot.extras.plugins import PluginFilterExtension
+from nautobot.extras.plugins import FilterExtension
```

Comment:
Since this is already being changed, shouldn't imports like this come from `nautobot.apps` instead?
```suggestion
from nautobot.apps.filters import FilterExtension
```

See: https://docs.nautobot.com/projects/core/en/stable/code-reference/nautobot/apps/filters/#nautobot.apps.filters.FilterExtension